Portal/View/Component.swift

```diff
@@ -141,8 +141,8 @@ extension Component {
         case .mapView(let properties, let style, let layout):
             return .mapView(properties, style, layout)
 
-        case .imageView(let image, let style, let layout):
-            return .imageView(image, style, layout)
+        case .imageView(let image, let clipToBounds, let style, let layout):
```

Comment:
In fact, this is related because when you clip to bounds an image it clips the shadow too, so I add this associated value to configure this parameter. When `clipToBounds` is disabled, the shadow works on an image. If it's always enabled, it will never work directly onto it. I think this has to be on this PR. I will add this to the base stylesheet, because this is a UIView property.